docs/contributing/CONTRIBUTING.md

```diff
@@ -45,22 +45,22 @@ extension](#running--testing-extension-locally). You may also find the [VSCode A
 
    where `some.extension` is the id of the extension you'd like to install into the sandbox
 
-6. Copy / symlink `cursorless-talon-dev` into your Talon user directory for some useful voice commands for developing Cursorless.
+6. Copy / symlink [`cursorless-talon-dev`](https://github.com/cursorless-dev/cursorless/tree/main/cursorless-talon-dev) into your Talon user directory for some useful voice commands for developing Cursorless.
```

Comment:
We generally use relative links when they're inside the repo

Reply:
I also prefer relative links. but when I look through the code base we haven't really been too rigorous about that unfortunately.

Reply:
I can believe that, but I don't that's an argument against changing this to a relative link, no?

Reply:
Nope